kvs-key-value-pairs/kvs-key-value-pairs.js

```diff
@@ -0,0 +1,27 @@
+import cf from 'cloudfront';
+​
+// (Optional) Declare the ID of the key value store that you have associated with this function
```

Comment:
I would remove this and replace it with a comment about remembering to attach the KVS to your function before referencing KVS in your code.